docker-compose.yml

```
@@ -0,0 +1,117 @@
+    restart: unless-stopped
+    networks:
+      - saleor-backend-tier
+    stdin_open: true
+    tty: true
+    depends_on:
+      - db
+      - redis
+      - jaeger
+    volumes:
+      # shared volume between worker and api for media
+      - saleor-media:/app/media 
```

yamllint

[error] trailing spaces

(trailing-spaces)